src/main/java/edu/harvard/iq/dataverse/marketplace/controller/api/UsersController.java

```diff
@@ -0,0 +1,63 @@
+package edu.harvard.iq.dataverse.marketplace.controller.api;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import edu.harvard.iq.dataverse.marketplace.model.User;
+import edu.harvard.iq.dataverse.marketplace.openapi.annotations.AuthAPIDocs;
+import edu.harvard.iq.dataverse.marketplace.payload.ServerMessageResponse;
+import edu.harvard.iq.dataverse.marketplace.payload.auth.UserDTO;
+import edu.harvard.iq.dataverse.marketplace.repository.UserRepo;
+import edu.harvard.iq.dataverse.marketplace.security.ApplicationRoles;
+
+
+@RestController
+@RequestMapping("/api/user")
```

Comment:
```suggestion
@RequestMapping("/api/users")
```

I think users might be a bit better. It's what GitHub uses, for example:

`https://api.github.com/users?since=XXX`

Just like how we have /api/auth/roles to get a list of roles.